src/plugins/intel_cpu/tests/functional/subgraph_tests/src/fullyconnected_sparse.cpp

```diff
@@ -0,0 +1,329 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
I am convinced that it is better to extend existing CPU single layer test instead of separate one.
Please extract the test into separate PR to speedup merge of production functionality.

Reply:
will be done in separate PR, thanks